tests/app/submitter/test_converter.py

```diff
@@ -182,6 +182,7 @@ def test_converter_language_code_not_set_in_payload(
         fake_questionnaire_schema, fake_questionnaire_store, {}, SUBMITTED_AT
     )
 
+    assert fake_questionnaire_store.metadata.get("launch_language_code") is None
```

Comment:
This might not be worth worrying about, but this injects behaviour that you might not want - the presence of `launch_language_code` as `None` is different to it not existing and adding some defaulting behaviour in the test. Maybe:
```suggestion
    with pytest.raises('KeyError'):
        fake_questionnaire_store.metadata["launch_language_code"]
```

Reply:
Updated, but also realised key should be `language_code` so have changed that.